tests/unit/conftest.py

```diff
@@ -41,7 +40,7 @@ def mock_cli(
     # We need this to suppress warnings for operations that don't
     # have access to the cli.suppress_warnings attribute.
     # e.g. operation defaults
-    sys.argv.append("--suppress-warnings")
+    # sys.argv.append("--suppress-warnings")
```

Comment:
Maybe we can add a bit context on why it needs to be removed here for visibility.